dashboard/app/models/script.rb

```diff
@@ -855,7 +859,9 @@ def self.build_property_hash(script_data)
       hideable_stages: script_data[:hideable_stages] || false, # default false
       professional_learning_course: script_data[:professional_learning_course] || false, # default false
       peer_reviews_to_complete: script_data[:peer_reviews_to_complete] || nil,
-      student_detail_progress_view: script_data[:student_detail_progress_view] || false
+      student_detail_progress_view: script_data[:student_detail_progress_view] || false,
+      project_widget_visible: script_data[:project_widget_visible] || false,
+      project_widget_types: script_data[:project_widget_types] || nil
```

Comment:
nit: I don't think the `|| nil` does anything here. I expect `script_data[:project_widget_types]` to be an array or nil already. If it's an array, even an empty array, then `|| nil` will preserve it. Otherwise it's already nil.